apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx

```diff
@@ -133,7 +133,7 @@ const PlanUpdateSidePanel = () => {
         currentPlan: subscription?.plan?.name,
       }
       if (source) {
-        properties.upgradeOrigin = source
+        properties.origin = source
```

Comment:
nitpick, not calling it upgrade, since users technically haven't upgraded yet, this just leads to the plan update panel